# grasp_state can be either 0 for pregrasp, 1 for grasp and 2 for postgrasp
uint8 PRE_GRASP_STATE=0
uint8 GRASP_STATE=1
uint8 POST_GRASP_STATE=2
uint8 grasp_state
StandardisedGrasp grasp
float64 max_torque
